modules/tfm/tfm/boards/common/tfm_hal_platform.c

```diff
@@ -52,6 +52,26 @@ static enum tfm_hal_status_t crypto_platform_init(void)
 }
 #endif /* defined(TFM_PARTITION_CRYPTO) */
 
+/* To write into AIRCR register, 0x5FA value must be write to the VECTKEY field,
+ * otherwise the processor ignores the write.
+ */
+#define SCB_AIRCR_WRITE_MASK ((0x5FAUL << SCB_AIRCR_VECTKEY_Pos))
```

Comment:
Isn't the variable name a bit misleading? Its the key value, not a mask, right?

Reply:
I agree, it is a copy-pasted define from TF-M.